ts/src/bingx.ts

```diff
@@ -524,11 +524,11 @@ export default class bingx extends Exchange {
          * @returns {object} an associative dictionary of currencies
          */
         if (!this.checkRequiredCredentials (false)) {
-            return undefined;
+            return {};
```

Comment:
this will affect `setMarkets` 
![Uploading image.png…]()

Reply:
![image](https://github.com/user-attachments/assets/611dd3dc-7994-4e8b-9fb1-dca0b7689c05)